### PR TITLE
Add endpoint to check active invites

### PR DESCRIPTION
**Pull request by Nikolya9192**

### CODE REVIEW

**Review by AndriyKovalets — APPROVED** [review]

File: Provis.Core/DTO/userDTO/UserActiveInviteDTO.cs [hunk 1]
```diff
@@ -0,0 +1,7 @@
+﻿namespace Provis.Core.DTO.userDTO
```
Comment (YevheniiVovk) [inline Provis.Core/DTO/userDTO/UserActiveInviteDTO.cs:1]: Please change namespace to  .UserDTO

File: Provis.Core/DTO/userDTO/UserActiveInviteDTO.cs [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+{
+    public class UserActiveInviteDTO
```
Comment (YevheniiVovk) [inline Provis.Core/DTO/userDTO/UserActiveInviteDTO.cs:3]: Can this model be populated and extending by some user cases or we can just return true/false at controller

File: Provis.Core/Services/UserService.cs [hunk 3]
```diff
@@ -62,5 +62,20 @@ public async Task<List<UserInviteInfoDTO>> GetUserInviteInfoListAsync(string use
...
+        {
+            var user = await _userRepository.GetByKeyAsync(userId);
+
+            if (user == null)
+            {
+                throw new HttpException(System.Net.HttpStatusCode.NotFound, "User with Id not exist");
+            }
+            var userActiveInviteDTO = new UserActiveInviteDTO();
+
+            userActiveInviteDTO.IsActiveInvite = await _inviteUserRepository.Query().AnyAsync(u => u.ToUserId == userId && u.IsConfirm == null);
```
Comment (YevheniiVovk) [inline Provis.Core/Services/UserService.cs:76]: Could we encapsulate this query
To go forward with DRY

File: Provis.Core/DTO/userDTO/UserActiveInviteDTO.cs (hunk 1 above)
Comment (Nikolya9192) [reply Provis.Core/DTO/userDTO/UserActiveInviteDTO.cs:1]: Changed

**Review by Sergey988 — APPROVED** [review]

**Pull request merged**